changelog.d/2025.10.19.12.00.00.md

```
@@ -0,0 +1,2 @@
+## Fixed
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)